src/access-rules.js

```diff
@@ -1,10 +1,20 @@
 'use strict'
-
 let accessRules = {}
 
-accessRules.checkAccess = (identity, action) => {
-  // TODO: BASED ON SCHEMA.ACCESSRULES DEFINE IF THE IDENTITY CAN ACT AT THE MODEL
-  return null
+accessRules.checkAccess = (identity, model, action) => {
+  let operation = false
+  try {
+    operation = action.schema.accessRules(identity, model)[action.operation]
+  } catch (e) {
+    throw new Error('Operation not defined at schema.accessRules')
+  }
+  if (typeof operation === 'function') {
+    operation = operation(identity, model)
+  }
+  if (!operation) {
+    throw new Error('Operation not allowed in this model')
```

Comment:
This error message should be more meaningful... maybe saying which operation it tried to run.